src/schedule.py

```diff
@@ -103,7 +166,71 @@ def assign_upstream_path_lengths(graph):
     
     return graph
 
-def schedule(computation_graph, hw_element_counts):
+def schedule(graph, hw_element_counts):
```

Comment:
rename to sdc_schedule.
and call sdc_schedule from simulator and symbolic_simulator